ArrayList.java

```diff
@@ -0,0 +1,39 @@
+
+public class ArrayList<T> {
+    public Object[] arrayList;
+    public int size = 0;
+
+
+    public ArrayList(int size){
+          arrayList = new Object[size];
+    }
+
+    public void add (T added){
+
+        if (size == arrayList.length){
+            Object[] newArray = new Object[arrayList.length*2];
+            for (int i = 0; i < arrayList.length; i++) {
+                newArray[i] = arrayList[i];
+            }
+            newArray[arrayList.length] = added;
+            arrayList = newArray;
+        }
+        arrayList[size] = added;
+        size++;
+
+    }
```

Comment:
```suggestion
    public void add (T added){
        if (size == arrayList.length){
            Object[] newArray = new Object[arrayList.length*2];
            for (int i = 0; i < arrayList.length; i++) {
                newArray[i] = arrayList[i];
            }
            newArray[arrayList.length] = added;
            arrayList = newArray;
        }
        arrayList[size] = added;
        size++;
    }
```
[formatting] no need for enters at the beginning/end of a func